server/utils/users/data/rolesToPermissions.json

```diff
@@ -0,0 +1,186 @@
+      "cas1_premises_view",
+      "cas1_request_for_placement_withdraw_others",
+      "cas1_space_booking_list",
+      "cas1_space_booking_view",
+      "cas1_space_booking_withdraw",
+      "cas1_user_list",
+      "cas1_view_cru_dashboard",
+      "cas1_view_manage_tasks",
+      "cas1_space_booking_create",
+      "cas1_out_of_service_bed_create_bed_on_hold",
+      "cas1_out_of_service_bed_cancel",
+      "cas1_out_of_service_bed_create",
+      "cas1_view_out_of_service_beds"
+    ]
+  },
+  {
+    "name": "cru_member_enable_out_of_service_beds",
+    "permissions": [
+      "cas1_out_of_service_bed_create",
+      "cas1_view_out_of_service_beds"
+    ]
+  },
+  {
+    "name": "change_request_dev",
+    "permissions": [
+      "cas1_appeal_create",
+      "cas1_planned_transfer_create"
+    ]
+  },
+  {
+    "name": "applicant",
+    "permissions": []
+  },
+  {
+    "name": "report_viewer",
+    "permissions": [
+      "cas1_reports_view"
+    ]
+  },
+  {
+    "name": "report_viewer_with_pii",
+    "permissions": [
+      "cas1_reports_view",
+      "cas1_reports_view_with_pii"
+    ]
+  },
+  {
+    "name": "excluded_from_assess_allocation",
+    "permissions": []
+  },
+  {
+    "name": "excluded_from_match_allocation",
+    "permissions": []
+  },
+  {
+    "name": "excluded_from_placement_application_allocation",
+    "permissions": []
+  },
+  {
+    "name": "appeals_manager",
+    "permissions": [
+      "cas1_assess_appealed_application",
+      "cas1_assess_application",
+      "cas1_process_an_appeal",
+      "cas1_view_assigned_assessments"
+    ]
+  },
+  {
+    "name": "janitor",
+    "permissions": [
+      "cas1_adhoc_booking_create",
+      "cas1_assess_appealed_application",
+      "cas1_assess_application",
+      "cas1_assess_placement_application",
+      "cas1_booking_create",
+      "cas1_booking_withdraw",
+      "cas1_booking_change_dates",
+      "cas1_out_of_service_bed_create",
+      "cas1_out_of_service_bed_create_bed_on_hold",
+      "cas1_out_of_service_bed_cancel",
+      "cas1_placement_request_record_unable_to_match",
+      "cas1_process_an_appeal",
+      "cas1_user_list",
+      "cas1_user_management",
+      "cas1_view_assigned_assessments",
+      "cas1_view_cru_dashboard",
+      "cas1_view_manage_tasks",
+      "cas1_view_out_of_service_beds",
+      "cas1_space_booking_create",
+      "cas1_space_booking_list",
+      "cas1_space_booking_record_arrival",
+      "cas1_space_booking_record_departure",
+      "cas1_space_booking_record_non_arrival",
+      "cas1_space_booking_record_keyworker",
+      "cas1_space_booking_view",
+      "cas1_space_booking_withdraw",
+      "cas1_premises_view",
+      "cas1_premises_manage",
+      "cas1_application_withdraw_others",
+      "cas1_reports_view",
+      "cas1_reports_view_with_pii",
+      "cas1_request_for_placement_withdraw_others",
+      "cas1_appeal_create",
+      "cas1_planned_transfer_create"
+    ]
+  },
+  {
+    "name": "user_manager",
```

Comment:
should this be `role_admin`? (just checking on the confluence)

Reply:
Oooh good catch, it's the other way round -- the Confluence page needs updating. I'll do that now!